client/src/pages/Contest/NewContest.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { FunctionComponent, Fragment } from 'react';
+import { NavLink } from 'react-router-dom';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import Paper from '@material-ui/core/Paper';
+import Box from '@material-ui/core/Box';
+import Link from '@material-ui/core/Link';
+import Grid from '@material-ui/core/Grid';
+import Container from '@material-ui/core/Container';
+import { FormikHelpers } from 'formik';
+import Typography from '@material-ui/core/Typography';
+import useStyles from './useStyles';
+import AuthHeader from '../../components/AuthHeader/AuthHeader';
+import NewContestForm from './NewContestForm/NewContestForm';
+//import { useAuth } from '../../context/useAuthContext';
+//import { useSnackBar } from '../../context/useSnackbarContext';
+
+const NewContest: FunctionComponent = (): JSX.Element => {
+  const classes = useStyles();
+
+  const handleSubmit = (
+    {
+      title,
+      description,
+      prizeAmount,
+      date,
+      time,
+      timezone,
+    }: {
+      title: string;
+      description: string;
+      prizeAmount: string;
+      date: string;
+      time: string;
+      timezone: string;
+    },
+    {
+      setSubmitting,
+    }: FormikHelpers<{
+      title: string;
+      description: string;
+      prizeAmount: string;
+      date: string;
+      time: string;
+      timezone: string;
+    }>,
+  ) => {
+    console.log('form sub', title.replace, description, prizeAmount, date, time, timezone);
+    setSubmitting(false);
+  };
+
+  return (
+    <Grid container component="main" className={classes.root}>
+      <CssBaseline />
+      <Grid container direction="column">
+        {/* <Grid container direction="row" alignItems="center" justify="space-between" className={classes.navigation}>
```

Comment:
Eventually you can remove this, as Jeri will be merging her Dashboard PR which has the navbar component.